evm_body.yaml

```diff
@@ -1708,3 +1708,126 @@ blockNumber_uncleIndex_param_l2:
       items:
         type: string
       default: ['latest', '0x0']
+
+# ======= zkEVM Methods =======
+
+zkevm_consolidatedBlockNumber:
+  allOf:
+    - $ref: '#/common_request_fields'
+    - type: object
+      properties:
+        method:
+          $ref: ./components/schemas.yaml#/Method
+          default: zkevm_consolidatedBlockNumber
+
+zkevm_isBlockVirtualized:
+  allOf:
+    - $ref: '#/common_request_fields'
+    - type: object
+      properties:
+        method:
+          $ref: ./components/schemas.yaml#/Method
+          default: zkevm_isBlockVirtualized
+        params:
+          type: array
+          description: |
+            1. String - The block number (in hex)
+          minItems: 1
+          maxItems: 1
+          items:
+            type: string
+          default: ['0x0']
+
+zkevm_isBlockConsolidated:
+  allOf:
+    - $ref: '#/common_request_fields'
+    - type: object
+      properties:
+        method:
+          $ref: ./components/schemas.yaml#/Method
+          default: zkevm_isBlockConsolidated
+        params:
+          type: array
+          description: |
+            1. String - The block number (in hex)
+          minItems: 1
+          maxItems: 1
+          items:
+            type: string
+          default: ['0x0']
```

Comment:
not super necessary but we can/should eventually add `pattern` to enforce the hex format

Reply:
Will add this @elanh

Reply:
Added @elanh